lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -685,9 +686,9 @@ def _section_data_download(course, access):
         settings.FEATURES.get('ENABLE_SPECIAL_EXAMS', False) and
         course.enable_proctored_exams
     )
-
+    section_key = 'data_download_2' if data_download_v2_is_enabled(course_key) else 'data_download'
```

Comment:
where is this section_key used exactly? Can you give the file reference?

Reply:
https://github.com/edx/edx-platform/blob/3879ad2252e474eefe87d43395ec27a05258be09/lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html#L121